lmvachon/bad.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
```

Comment:
I see we've got a closing `</html>` tag, but not an opening one. I'd add `<html lang="en">` here.